tensorflow/contrib/learn/python/learn/estimators/linear.py

```diff
@@ -243,8 +243,7 @@ def sdca_model_fn(features, labels, mode, params):
 
   parent_scope = "linear"
 
-  with variable_scope.variable_scope(
-          values=features.values(), name_or_scope=parent_scope) as scope:
+  with variable_scope.variable_scope(values=features.values(), name_or_scope=parent_scope) as scope:
```

Comment:
Hate to say it but there's also an 80 character line limit

Reply:
haha wow.

Reply:
Does anyone at Google use PyCharm? Because it is not a fan of this 😆

Reply:
Clearly you're being punished for not using Emacs :)

Reply:
People don't actually use that? I think this could go down as the least productive pull request of all time haha.